reference/errors/error_codes.mdx

```diff
@@ -126,6 +126,10 @@ The [`offset`](/reference/api/keys#query-parameters) parameter is invalid. It sh
 
 The given [`uid`](/reference/api/keys#uid) is invalid. The `uid` must follow the [uuid v4](https://www.sohamkamani.com/uuid-versions-explained) format.
 
+## `invalid_attributes_to_search_on`
```

Comment:
```suggestion
## `invalid_search_attributes_to_search_on`
```

related to https://github.com/meilisearch/documentation/issues/2478#issuecomment-1617782149

Reply:
Brilliant, thanks!